### PR TITLE
Implement project management

### PR DESCRIPTION
**Pull request by kkaneda**

### CODE REVIEW

**Comment by kkaneda** [comment]
Will add more unit test before merging.

File: server/internal/server/authorization.go [hunk 1]
```diff
@@ -109,6 +152,123 @@ func (s *Server) authorizedAPIKey(apiKey *cache.K, requestScope string) bool {
...
+		return projects[0].ID, nil
+	}
+
+	// When neither org ID nor project ID is specified, first project where the user belongs to and then org.
+
+	if len(userProjects) > 0 {
+		return userProjects[0].ProjectID, nil
+	}
+
+	log.Printf("userOrgs: %+v\n", userOrgs)
```
Comment (Ladicle) [inline server/internal/server/authorization.go:258]: how about changing the log level?

**Review by Ladicle — APPROVED** [review]
LGTM

File: server/internal/server/authorization.go (hunk 1 above)
Comment (kenji-cloudnatix) [reply server/internal/server/authorization.go:258]: Ah my bad. Will remove.

**Pull request merged**